app/controllers/application_controller.rb

```diff
@@ -17,4 +17,8 @@ def current_user
   def caseload
     sso_identity['caseload']
   end
+
+  def session_expired?
```

Comment:
What do you think about reversing the logic to make this `session_valid?`.  Just a thought.

Reply:
Yes good point - will change